scripts/e2e-test.sh

```diff
@@ -78,3 +78,24 @@ helm template deploy/helm/eunomia-operator/ \
   --set eunomia.operator.image.tag=dev \
   --set eunomia.operator.namespace=$TEST_NAMESPACE | kubectl delete -f -
 
+helm template deploy/helm/eunomia-operator/ \
+  --set eunomia.operator.image.tag=dev \
+  --set eunomia.operator.image.pullPolicy=Never \
+  --set eunomia.openshift.route.enabled=true | kubectl apply -f -
+
+kubectl apply -f examples/hello-world-yaml/test.yaml
+
+sleep 30s
```

Comment:
don't use sleep, try using `kubectl wait` similar as in l.65 instead

Reply:
Actually i tried  to use wait flag. 
But what's happening is CR is creating deployment, if i use wait flag its checking before CR is creating deployment and its failing with no deployment found. 
I believe  is wait  flag will work only  for existing resource at that point , in my case CR is taking 20s to become success . 
I tried using wait for GitOpsConfig to become ready . But it did not worked seems GitOpsConfig don't have that field to check wait flag.

Reply:
Thanks @akavel .
Removed sleep condition.